gazelle/testdata/simple_test_with_conftest/__test__.py

```diff
@@ -0,0 +1,12 @@
+import unittest
```

Comment:
is the `__test__.py` file a gazelle convention that automatically becomes the test entrypoint, or is it generated? Reason I'm asking is this file name doesn't seem to match the allowed test patterns in generate, but gets included in the py_test sources.

Reply:
It's special for Gazelle. The following is the main logic (though they are used in some other places).

https://github.com/bazelbuild/rules_python/blob/24a8b7460d03b7719d82a2ddfea794a0da4ac38d/gazelle/generate.go#L27-L28

https://github.com/bazelbuild/rules_python/blob/24a8b7460d03b7719d82a2ddfea794a0da4ac38d/gazelle/generate.go#L82-L83

https://github.com/bazelbuild/rules_python/blob/24a8b7460d03b7719d82a2ddfea794a0da4ac38d/gazelle/generate.go#L271-L275